RPi3.md

```diff
@@ -0,0 +1,278 @@
+# Raspberry Pi 3 Deployment with Pruning and FastSync
+
+Andreas Antonopoulos tells us "Not your keys, not your Bitcoin". Here's another Bitcoin axiom: "Your hardware. Your node. Your keys. Your Bitcoin."
+
+For your Bitcoin experience to be truly self-sovereign and trustless, you should run nodes on your own hardware and internet connection. BTCPayServer is an excellent way to run Bitcoin & Lightning nodes together. Not only are you validating transactions, you also get the ability to accept Bitcoin (layer 1) and Lightning (layer 2) payments.
+
+To that end, here are instructions to install and host your very own BTCPayServer on a Raspberry Pi. We call it a BTCPi.
```

Comment:
Maybe we can use this introduction sentence to stretch the fact we are talking about the RPI3 here. We should also mention, that the RPI4 is the recommended way to go and lprovide a link to the other docs, similar to the RPI4 page linking to this one.